### ISSUE TITLE
Stop Working!!!!!!!

### ISSUE THREAD

**Issue by Msameim181** [manojVivek/medium-unlimited#167]
Suddenly stop working and no more stories. I even updated to 1.10, but nothing changed.

![image](https://user-images.githubusercontent.com/25700939/126031591-6ab387c7-48c0-4d72-9845-5fbc0d3af7b9.png)

**Comment by ardzz** [comment]
same here

**Comment by julien040** [comment]
I'm not the only one as I can see

**Comment by ntvinhit** [comment]
Medium was updated :(

**Comment by ghost** [comment]
Outline.com stopped working as well. Hoping they didn't block it on backend...

**Comment by julien040** [comment]
If you disconnect from your account and refresh, it can work

**Comment by rafikta** [comment]
Yes, i had the same problem, signed out and cleared medium.com cookies and refresh the page and works.

**Comment by ace3** [comment]
i have luck by append the URL with `https://medium.com/m/global-identity?redirectUrl=<medium content url>` ,

**Comment by phimanh2905** [comment]
I was signed out and cleared medium.com cookies and refresh the page and works but NOT WORK. Any one help me?
this link: https://blog.angulartraining.com/ngrxlet-a-better-version-of-the-async-pipe-37bfc5c7a5ec

**Comment by robert-tsai-007** [comment]
@phimanh2905  me,too!! looks like this one right?
**
![image](https://user-images.githubusercontent.com/88974400/129481080-24ac7b0b-26a0-4e92-92fe-8cf22a8eb993.png)
**

**Comment by vaibhavkumar-sf** [comment]
Same problem

**Comment by d9k** [comment]
@manojVivek, plz check

**Comment by Msameim181** [comment]
1. I signed out 
2. Refresh all pages
3. Clear all the cookies from medium or towards data science, etc.
4. Sign in to medium
5. Refresh all pages

I tried and it seems worked

**Comment by vaibhavkumar-sf** [comment]
WooHoo .. I just cleared cookie and signed-in again .. and it worked fine. Now it is opening full article.. 

https://preettheman.medium.com/vs-code-just-got-a-massive-update-658aee5fcf3b

Update ::
Above trick just worked for 3 articles, after that it started trimming content again

**BETTER WORKING TRICK :**
: Read  3-5 articles in INCOGNITO MODE of chrome (No need to login)
: After that it will ask for login, simply close incognito mode of chrome, 
: then Open INCOGNITO MODE again, you can again read 3-5 articles without login. 

I tried same trick multiple times.

**Comment by IamAdiSri** [comment]
I've tried all of the solutions suggested in this thread, none of which have worked for me. Definitely feels like there's some problem with the add on.

Edit: Ofc you can use the Incognito mode work around, but the whole point of the add on was to not have to resort to that...

**Comment by mewforest** [comment]
Is there any alternatives for Chrome?

**Comment by vaibhavkumar-sf** [comment]
Copy the URL which is not opening completely in normal browser, then open that url in incognito mode of chrome, press **CTRL + SHIFT + N** to open incognito mode.

**Comment by 470452326** [comment]
Same problem

**Comment by AyushRawal** [comment]
> Yes, i had the same problem, signed out and cleared medium.com cookies and refresh the page and works.

I think this should be added to the README. This is a very common issue.

**Comment by htr3n** [comment]
If I must clear cookies to make this plugin work, then I wouldn't need it in the first place because I have used that method (clearing cookies, for instance, this excellent plugin: https://github.com/dessant/clear-browsing-data) before using this.

**Comment by vasiauvi** [comment]
Unfortunately no solution provided above worked for me. It's seems that the extension is not effective now on Medium, maybe the site has some improvements.

**Comment by alphavector** [comment]
In fact, the articles open fine in incognito mode, so @Nikostito's changes were correct, they just need to be refined
